x-pack/plugins/snapshot_restore/__jest__/client_integration/repository_edit.test.ts

```diff
@@ -215,7 +215,7 @@ describe('<RepositoryEdit />', () => {
       );
       expect(find('readOnlyToggle').props()['aria-checked']).toBe(settings.readonly);
 
-      const codeEditor = testBed.component.find('EuiCodeEditor');
+      const codeEditor = testBed.component.find('EuiCodeEditor').at(1);
```

Comment:
Because we're wrapping EuiCodeEditor within a lazy-loading component of the same name, we need to access the second one found.

Reply:
nit: would it be possible to use a `data-test-subj` here?

Reply:
I don't believe we can use a `data-test-subj` here. That will retrieve an element from the virtual DOM, but in this case we want to retrieve an Enzyme ReactWrapper instance so we can [access its props](https://github.com/elastic/kibana/blob/d9c3a24d19e9e1aa2940352d1ec76cb5b0f77c75/x-pack/plugins/snapshot_restore/__jest__/client_integration/repository_edit.test.ts#L219).

Reply:
For information, `find('my-test-subj')` does return a ReactWrapper instance 😊